### PR TITLE
{bio}[foss/2018a] BEDTools v2.28.0

### PR DESCRIPTION
**Pull request by ghuls**
This should fix issue #7951

### CODE REVIEW

**Comment by boegel** [comment]
(close/re-open to re-trigger Travis after target branch was changed to `develop`)

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+buildopts = 'CXX="$CXX"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
+
+files_to_copy = ["bin", "docs", "data", "genomes", "scripts", "test"]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['annotateBed', 'bamToBed', 'bamToFastq', 'bed12ToBed6', 'bedpeToBam', 'bedToBam', 'bedToIgv', 'bedtools', 'closestBed', 'clusterBed', 'complementBed', 'coverageBed', 'expandCols', 'fastaFromBed', 'flankBed', 'genomeCoverageBed', 'getOverlap', 'groupBy', 'intersectBed', 'linksBed', 'mapBed', 'maskFastaFromBed', 'mergeBed', 'multiBamCov', 'multiIntersectBed', 'nucBed', 'pairToBed', 'pairToPair', 'randomBed', 'shiftBed', 'shuffleBed', 'slopBed', 'sortBed', 'subtractBed', 'tagBam', 'unionBedGraphs', 'windowBed', 'windowMaker']],
```
Comment (boegel) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018a.eb]: @ghuls Do you mind breaking wrapping this line to have lines that are maximally 120 characters wide?

Something like:

```python
sanity_check_paths = {
    'files': ['bin/%s' % x for x in ['annotateBed', 'bamToBed', 'bamToFastq', 'bed12ToBed6',
                                     'bedpeToBam',, 'bedToBam', 'bedToIgv', 'bedtools', 'closestBed', 
                                     'clusterBed', 'complementBed', , 'coverageBed', ...]],
    'dirs': files_to_copy,
}

**Comment by vanzod** [comment]
@ghuls Previous versions of `BEDTools` required `BamTools` as a dependency, which is now missing here. Is this dependency not required anymore?

**Comment by boegel** [comment]
@vanzod Does #7951 answer your question?

**Comment by ghuls** [comment]
@vanzod I am not sure how much it still uses from Bamtools now that they use htslib. They include their own copy anyway: https://github.com/arq5x/bedtools2/tree/master/src/utils/BamTools

**Comment by vanzod** [comment]
@boegel It was not 100% clear to me from that issue. Plus it is useful to have an explicit reference for a dependency change directly into the PR for the love of our future selves.

@ghuls Ok, so the fact that they still include it does it mean that it is still needed? We always try to avoid internal builds and prefer to use components built by EasyBuild directly instead.

**Comment by boegel** [comment]
The toolchain being used here is now deprecated, and we have more recent versions of `BEDTools` already merged, so I'll close this PR; sorry for not actively following up on this @ghuls!

**Pull request closed without merging**